app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -258,6 +258,18 @@ class MessagingManager @Inject constructor(
             NotificationItem(0, now, jsonData[MESSAGE].toString(), jsonObject.toString(), source)
         notificationDao.add(notificationRow)
 
+        mainScope.launch {
+            if (integrationUseCase.getNotificationReceivedEvent()) {
+                try {
+                    integrationUseCase.fireEvent("mobile_app_notification_received", jsonData)
```

Comment:
For consistency with other [app events](https://companion.home-assistant.io/docs/integrations/app-events) and the mentioned HTML5 event, suggestion: `mobile_app.notification_received`.

(I know there is `mobile_app_notification_action`, but that is the exception!)

Reply:
There is also `mobile_app_notification_cleared` 😉

Reply:
OK, another one that's different, but there are still more events with a `.` 😉 For consistency with other notification events `_` like you did might make more sense, feel free to mark this as resolved if you want.

> we don't list any of the notification events here

Yeah I don't really understand why, they're still events sent by the app?

Reply:
I'm not sure why I didn't add them when I updated that page lol.